packages/contracts/contracts/voting/majority/MajorityVotingBase.sol

```diff
@@ -259,35 +288,17 @@ abstract contract MajorityVotingBase is
             return false;
         }
 
-        uint256 totalSupportPct = _calculatePct(proposal_.yes, proposal_.totalVotingPower);
-
-        // EARLY EXECUTION (after the vote start but before the vote duration has passed)
-        // The total support must greater than the relative support threshold (assuming that `relativeSupportThresholdPct > 50 >= totalSupportThresholdPct`).
-        if (_isVoteOpen(proposal_) && totalSupportPct > proposal_.relativeSupportThresholdPct) {
-            return true;
-        }
-
-        // NORMAL EXECUTION (after the vote duration has passed)
-        // Both, the total and relative support must be met.
-
-        // Criterium 1: The vote has ended.
         if (_isVoteOpen(proposal_)) {
-            return false;
-        }
-
-        // Criterium 2: The total support is greater than the total support threshold
-        if (totalSupportPct <= proposal_.totalSupportThresholdPct) {
-            return false;
+            // Early execution
+            return
+                worstCaseSupport(_proposalId) > proposal_.supportThreshold &&
+                participation(_proposalId) > proposal_.participationThreshold;
```

Comment:
Is participation a strict "greater than" operand?

Reply:
Changed `participationThreshold` to `minParticipation` in aa72391bb842e8672bc30a8a77a6ffb7f18a7c17.

Reply:
I don't mean about the names, I mean about the operand being used. We seem to use `>` in both while only one of them should